azure-pipelines-template.yml

```diff
@@ -42,10 +42,9 @@ jobs:
   - script: npm run prebuild
     displayName: Test prebuild
 
-  - ${{ if startsWith(variables['Build.SourceBranch'], 'refs/tags/v') }}:
```

Comment:
Why do we need to publish the artifacts if we're not on a release tag?

Reply:
we don't have to, just seemed useful to always store it in azure devops in case we want to download the prebuilds and test it or something

Reply:
oh ok, sure.